Directory.Build.props

```diff
@@ -42,6 +43,7 @@
   <PropertyGroup Condition=" Exists('$(MSBuildProjectDirectory)\version.json') ">
     <EnableNerdbank_GitVersioning Condition=" '$(EnableNerdbank_GitVersioning)' == '' ">true</EnableNerdbank_GitVersioning>
   </PropertyGroup>
+<UsingTask TaskName="Microsoft.Build.Tasks.AssignProjectConfiguration" AssemblyFile="$(MSBuildBinPath)\Microsoft.Build.Tasks.Core.dll" />
```

Comment:
oops I didn't realize I pushed that file.  That wasn't suppose to be included.  Good catch.  Bad on me for not seeing it.